src/utils/StorageManager.ts

```diff
@@ -19,29 +19,24 @@ import { logger } from "matrix-js-sdk/src/logger";
 
 import SettingsStore from "../settings/SettingsStore";
 import { Features } from "../settings/Settings";
+import { MigrationState } from "../../../matrix-js-sdk/src/crypto/store/base";
 
 const localStorage = window.localStorage;
 
-// just *accessing* indexedDB throws an exception in firefox with
-// indexeddb disabled.
-let indexedDB: IDBFactory;
-try {
-    indexedDB = window.indexedDB;
-} catch (e) {}
+// make this lazy in order to make testing easier
+function getIndexedDb(): IDBFactory | undefined {
```

Comment:
Not sure how to do that better. If it's not done like that it makes it very difficult to test in unit test (`StorageManager-test`) because fake indexed db wont work